ospfd/ospf_vty.c

```diff
@@ -564,6 +622,10 @@ DEFUN (no_ospf_network_area,
   VTY_GET_IPV4_PREFIX ("network prefix", p, argv[idx_ipv4_prefixlen]->arg);
   VTY_GET_OSPF_AREA_ID (area_id, format, argv[idx_ipv4_number]->arg);
 
+  if (IS_DEBUG_OSPF_TRACE)
```

Comment:
This was for internal daemon activity log, function name print provides context to what cli is executed with parameters